Linux Building - Gui

**Describe the bug**
running ```electronize start``` from the Gui directory yields the following error
```
user@ubuntu:~/Desktop/AttackSurfaceAnalyzer/Gui$ electronize start
Start Electron Desktop Application...
Microsoft (R) Build Engine version 16.0.450+ga8dc7f1d34 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.
/home/user/Desktop/AttackSurfaceAnalyzer/Gui/AttackSurfaceAnalyzerGui.csproj : error MSB4236: The SDK 'Microsoft.NET.Sdk.web' specified could not be found.
Error occurred during dotnet publish: 1
```

A small amount of searching yields a hint at the [OmniSharp project](https://github.com/OmniSharp/omnisharp-vscode/issues/2816)

For case-sensitive OS the ```AttackSurfaceAnalyzerGui.csproj``` needs an uppercase W in web
so changing
```
<Project Sdk="Microsoft.NET.Sdk.web">
```
to
```
<Project Sdk="Microsoft.NET.Sdk.Web">
```
makes the build go.

**To Reproduce**
Steps to reproduce the behavior:
1. Follow build instructions at https://github.com/Microsoft/AttackSurfaceAnalyzer/blob/master/BUILD.md on Ubuntu Linux

**Expected behavior**
```electrionize start``` doesnt fail

**Screenshots**
N/A

**System Configuration (please complete the following information):**
 - OS: Ubuntu 16.04
 - OS Version: Ubuntu 16.04.6
 - Application Version: N/A
 - CLI or GUI: Gui

**Additional Context**
Add any other context about the problem here.

--- a/Gui/AttackSurfaceAnalyzerGui.csproj
+++ b/Gui/AttackSurfaceAnalyzerGui.csproj
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.Default.props" Condition="Exists('$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.Default.props')" />
 <PropertyGroup>
     <Authors>Microsoft Corporation</Authors>

